GuessingGame.html

```diff
@@ -1,3 +1,4 @@
+<<<<<<< Updated upstream
 <script type="text/javascript">
```

Comment:
This looks like a merge conflict that you committed here. Were you able to get it fixed?